scripts/runtests.py

```diff
@@ -631,7 +651,7 @@
 
 def _start_output(msg):
     if sys.stdout.isatty():
-        sys.stdout.write('\r' + ' ' * 78 + '\r' + msg)
+        sys.stdout.write('\r' + ' ' * 120 + '\r' + msg)
```

Comment:
Hmm I want to keep the number of spaces as is. I think we need to abbreviate the status message when a line is too long. If you want, I can do it by myself.

Reply:
I just reverted this part. I left the abbreviation of the status message as future work.